auslib/db.py

```diff
@@ -15,6 +15,9 @@
 import migrate.versioning.schema
 import migrate.versioning.api
 
+from dictdiffer import merge, patch
```

Comment:
Nit: merge and patch are not terribly descriptive, so it would be part to just import dictdiffer and call them as dictdiffer.merge/patch.